skimage/filters/rank/tests/test_rank.py

```diff
@@ -1,11 +1,13 @@
+import os
```

Comment:
It looks like `os` is imported but not used.

Reply:
fixed